src/presentation/components/address-amount-form.tsx

```diff
@@ -150,7 +150,7 @@ const AddressAmountEnhancedForm = withFormik<AddressAmountFormProps, AddressAmou
         .required('Please enter a valid amount')
         .min(getMinAmountFromPrecision(props.assetPrecision), 'Amount should be at least 1 satoshi')
         .test('too-many-digits', 'Too many digits', (value) => {
-          return value !== undefined && value.toString().length < 14;
+          return value !== undefined && value.toString().length < 20;
```

Comment:
Thinking about, we know we cant have more that 21M of BTC, thats fine. But we cant be sure for the same thing ofr other assets that can have much more, potentially infinite big.

Shall we remove this check entirely?